Eatery Blue/Eatery Blue/UI/HomeScreen/HomeModelController.swift

```diff
@@ -259,7 +259,7 @@ class HomeModelController: HomeViewController {
         Task {
             await withTaskGroup(of: Void.self) { [self] group in
                 group.addTask {
-                    await updateAllEateriesFromNetworking()
+                    await self.updateAllEateriesFromNetworking()
```

Comment:
Pretty sure you can drop the `self.` since `self` is explicitly captured

```suggestion
                    await updateAllEateriesFromNetworking()
```

Reply:
I was getting a warning for not having the self but I can fix it